docs/database/database-spans.md

```diff
@@ -478,6 +480,39 @@ Semantic conventions for individual database systems or specialized instrumentat
 MAY specify a different `db.query.summary` format as long as produced summary remains
 relatively short and its cardinality remains low comparing to the `db.query.text`.
 
+## Context propagation
+
+**Status**: [Development][DocumentStatus]
+
+### SQL commenter
+
+Instrumentations MAY propagate context using [SQL commenter](https://google.github.io/sqlcommenter/spec/) by injecting comments into SQL queries before execution. Context propagation SHOULD NOT be enabled by default, but instrumentation MAY allow users to opt into it.
+
+The instrumentation implementation SHOULD **append** the comment to the end of the query. Semantic conventions for individual database systems MAY specify different format depending on the specific database system's requirements or preferences. For instance, the instrumentation MAY **prepend** the comment at the beginning of the query.
+
+The instrumentation SHOULD allow users to pass a propagator to overwrite the global propagator.
```

Comment:
FWIW, there's some prior art in Java Instrumentation of pairing a specific instrumentation to a specific propagator:

https://github.com/search?q=repo%3Aopen-telemetry%2Fopentelemetry-java-instrumentation%20AwsXrayPropagator.getInstance()&type=code

Maybe it's ok to pair this behavior directly to the w3c propagator (only supporting w3c trace context)?

Reply:
Do you mean that instead of using the global propagator by default, we should use the `W3C propagator` as the default one?

I don't have a preference here. What makes the `W3C propagator` the default, a better choice than the global propagator?

Reply:
Good point